.semaphore/semaphore.yml

```diff
@@ -62,6 +63,7 @@ blocks:
 
         - name: "forbidden api checks"
           commands:
+            - ${MVN} compile ${MAVEN_SKIP} ${MAVEN_SKIP_TESTS}
```

Comment:
why not test-compile here as well since we also check test code below?

Reply:
Fixed